src/generate.jl

```diff
@@ -46,20 +46,16 @@ function project(pkg::String, dir::String; preview::Bool)
         end
     end
 
-    authorstr = "[\"$name " * (email == nothing ? "" : "<$email>") * "\"]"
+    authors = ["$name " * (email == nothing ? "" : "<$email>")]
 
     uuid = UUIDs.uuid1()
     genfile(pkg, dir, "Project.toml"; preview=preview) do io
-        print(io,
-            """
-            authors = $authorstr
-            name = "$pkg"
-            uuid = "$uuid"
-            version = "0.1.0"
-
-            [deps]
-            """
-        )
+        toml = Dict("authors" => authors,
+                    "name" => pkg,
+                    "uuid" => string(uuid),
+                    "version" => "0.1.0",
+                    )
+        TOML.print(io, toml, sorted=true, by=key -> (Types.project_key_order(key), key))
```

Comment:
Is this a related change?

Reply:
Yes, the reason the user hit the bug in the first place was that we were not properly escaping the authors field on `generate`. I think their git name had a backslash in it, which is not properly escaped if we just splice it in directly.

Reply:
I can make them separate commits if necessary

Reply:
No, that's cool, makes sense. I was just wondering about the connection.